classes/class-uagb-admin.php

```diff
@@ -318,6 +318,15 @@ public static function activate_widget() {
 			$blocks[ $block_id ] = $block_id;
 			$blocks              = array_map( 'esc_attr', $blocks );
 
+			if ( 'how-to' === $block_id ) {
+				foreach ( $blocks as $slug => $value ) {
+					if ( 'info-box' === $slug && 'info-box' !== $value ) {
```

Comment:
Instead of the foreach we can simply write this:

```suggestion
					if ( 'how-to' === $block_id && 'disabled' === $blocks[ 'info-box' ] ) {
					      $blocks[ 'info-box' ] = 'info-box' ;
					}
```

Reply:
Right! Removed foreach